source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -24,6 +24,18 @@ std::atomic<bool>& deprecatedNameLogged() {
   MUTABLE_CONSTRUCT_ON_FIRST_USE(std::atomic<bool>, false);
 }
 
+/**
+ * Constant values used for tracing metadata.
+ */
+struct TracingConstantValues {
+  const std::string Status = "lua_http_call_status";
+  const std::string HttpStatus = "lua_http_call_http_status";
+  const std::string Success = "Success";
+  const std::string Failure = "Failure";
+};
+
+using TracingConstants = ConstSingleton<TracingConstantValues>;
```

Comment:
Do you really need this singleton? This seems just scoped to this file? Looks like you can juts use `constexpr absl::string_view` in the anonymous namespace.